src/pages/login/index.js

```diff
@@ -8,10 +8,39 @@ import './login.css';
 const Login = () => {
   const { onLogin } = useAuth();
   const [formData, setFormData] = useState({ email: '', password: '' });
+  const [emailIsValid, setEmailIsValid] = useState(false);
+  const [passwordIsValid, setPassordIsValid] = useState(false);
+
+  const isValidEmail = () => {
```

Comment:
see comment for password, please move the validation logic in a utils / helper function so that it can be reused